packages/next/src/executors/build/build.impl.ts

```diff
@@ -77,7 +81,21 @@ export default async function buildExecutor(
     mkdir(options.outputPath);
   }
 
```

Comment:
Cleanup to be in sync with other parts of the system. Instead of encapsulated `createPackageJson` we have now `updatePackageJson` than makes additional changes to JSON before we save it and feed it to lock file creator.